packages/styles/_typography.scss

```diff
@@ -19,9 +19,17 @@
 // For each size in the $type-sizes variable
 // Create class (.mono.xxl)
 @each $name, $size in $type-sizes {
-  .mono.#{$name} {
+  .mono.type-#{$name} {
     font-size: calc(var(--type-#{$name}) * .95);
   }
 }
 
-
+/*  Colors
+========================================================================== */
+// For each color in $colors variable
+// Create class (.color-blue-base)
+@each $name, $color in $colors {
+  .color-#{$name} {
+    color: var(--#{$name})
```

Comment:
can you add a comment in utilities.md that `.color-{name}` is a utility class? Also would be nice to cross link in `colors.md`

```md
### Colors
| Class                   | Properties | Example
| :---------------------- |:-----------|:-----------
| .color-{color} |  color: var(--{color}); | `class="color-blue-base"`
```